cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANBackendImpl.java

```diff
@@ -566,4 +691,45 @@ private JsonResponse doCKANRequest(String method, String urlPath, String jsonStr
         return doRequest(method, urlPath, true, headers, new StringEntity(jsonString, "UTF-8"));
     } // doCKANRequest
 
+    public class DataStore{
```

Comment:
The idea for this class is to be the same as the method `createDataStore`? In that case, not sure it should be a class or an inner class for this backend. 

Why do you think it's better to abstract it as a new class?

Reply:
Well, I am trying to use the same structure that is used in cygnus-ngsi for creating Json from Gson object, this is a similar approach presented in the class NotifyContextRequest, this is also outdated?

Reply:
Not really. I'm not sure if that class should be an inner class. In the NotifyContextRequest class makes sense because NotifyContextRequest defines an object. 

In this case CKANBackendImpl is not a model. Please refactor it and create a new package to put it in.

cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/ckan/model 

@fgalan do you agree?

Reply:
> @fgalan do you agree?

I agree.

Reply:
Done, please check 5c6e2fb

Reply:
Great! Thanks.

Reply:
Thank you for the advice